### PR TITLE
Update release notes + License dependencies for 6.1.0 & 6.1.1 releases

### PR DESCRIPTION
**Pull request by ajainarayanan**
Quick build: https://builds.cask.co/browse/CDAP-DQB465

### CODE REVIEW

**Comment by ajainarayanan** [comment]
Docs build page: https://builds.cask.co/artifact/CDAP-DQB465/shared/build-2/Docs-HTML/html/reference-manual/licenses/index.html

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -30,6 +30,141 @@ Cask Data Application Platform Release Notes
...
+- :cask-issue:`CDAP-15659` - Route user directly to the pipeline's detail page from pipeline card in Control Center. 
+
+- :cask-issue:`CDAP-15489` - New user experience for log level selection.
+
+- :cask-issue:`CDAP-15265` - Added image version as a configuration setting to the dataproc provisioner
+
+- :cask-issue:`CDAP-16076` - Improved the way pipelines with macros that are provided by intermediate stages run.
+
+
+Deprecation
```
Comment (sreevatsanraman) [inline cdap-docs/reference-manual/source/release-notes.rst]: Can this be moved to the top? Easy to miss deprecations

**Review by sreevatsanraman — CHANGES_REQUESTED** [review]
Please address the one feedback and merge

**Comment by ajainarayanan** [comment]
https://builds.cask.co/artifact/CDAP-DQB465/shared/build-4/Docs-HTML/html/reference-manual/release-notes.html#summary Latest docs.

**Review by sagarkapare — APPROVED** [review]

**Review by sreevatsanraman — APPROVED** [review]

**Pull request merged**